db/migrate/20221124084859_add_omniauth_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddOmniauthToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column(:users, :provider, :string, limit: 50, null: false, default: "")
+    add_column(:users, :uid, :string, limit: 50, null: false, default: "")
```

Comment:
Is `uid` an omniauth field, and if yes, what does it mean? Is this some ID we get from OIDC?

Reply:
Yes, `uid` is an omniauth field AFAICT. Most likely the field refers to the external `uuid` of the HPI user's account.